src/libraries/Microsoft.PowerFx.Core/Types/Enums/EnumStore.cs

```diff
@@ -198,8 +198,10 @@ internal class EnumStore
                 // TASK 4620228: Connect to Data: Need Final design on Error enum returned by Errors function.
                 {
                     EnumConstants.ErrorKindEnumString,
-                    "%n[None:0, Sync:1, MissingRequired:2, CreatePermission:3, EditPermission:4, DeletePermission:5, Conflict:6, NotFound:7, ConstraintViolated:8, GeneratedValue:9, ReadOnlyValue:10, Validation: 11, Unknown: 12, " +
-                    "Div0: 13, BadLanguageCode: 14, BadRegex: 15, InvalidFunctionUsage: 16, FileNotFound: 17, AnalysisError: 18, ReadPermission: 19, NotSupported: 20, InsufficientMemory: 21, QuotaExceeded: 22, Network: 23, Numeric: 24, InvalidArgument: 25]"
+                    "%n[None:0, Sync:1, MissingRequired:2, CreatePermission:3, EditPermission:4, DeletePermission:5, Conflict:6, NotFound:7, ConstraintViolated:8, " +
+                    "GeneratedValue:9, ReadOnlyValue:10, Validation: 11, Unknown: 12, Div0: 13, BadLanguageCode: 14, BadRegex: 15, InvalidFunctionUsage: 16, " +
+                    "FileNotFound: 17, AnalysisError: 18, ReadPermission: 19, NotSupported: 20, InsufficientMemory: 21, QuotaExceeded: 22, Network: 23, Numeric: 24, " +
+                    "InvalidArgument: 25, Canceled: 26]"
```

Comment:
The numeric values here do not match those from ErrorKind above; i.e. there is no ErrorKind.InvalidArgument and ErrorKind.Canceled is 25, not 26.  Is that intentional?

Reply:
Good catch; InvalidArgument should be there. Fixing it.